src/main/kotlin/com/terraformation/backend/search/table/ParticipantProjectSpeciesTable.kt

```diff
@@ -0,0 +1,49 @@
+package com.terraformation.backend.search.table
+
+import com.terraformation.backend.auth.currentUser
+import com.terraformation.backend.db.accelerator.ParticipantProjectSpeciesId
+import com.terraformation.backend.db.accelerator.tables.references.PARTICIPANT_PROJECT_SPECIES
+import com.terraformation.backend.db.default_schema.tables.references.PROJECTS
+import com.terraformation.backend.db.default_schema.tables.references.SPECIES
+import com.terraformation.backend.search.SearchTable
+import com.terraformation.backend.search.SublistField
+import com.terraformation.backend.search.field.SearchField
+import org.jooq.Condition
+import org.jooq.Record
+import org.jooq.TableField
+import org.jooq.impl.DSL
+
+class ParticipantProjectSpeciesTable(tables: SearchTables) : SearchTable() {
+  override val primaryKey: TableField<out Record, out Any?>
+    get() = PARTICIPANT_PROJECT_SPECIES.ID
+
+  override val sublists: List<SublistField> by lazy {
+    with(tables) {
+      listOf(
+          projects.asMultiValueSublist(
+              "projects", PARTICIPANT_PROJECT_SPECIES.PROJECT_ID.eq(PROJECTS.ID)),
+          species.asMultiValueSublist(
+              "species", PARTICIPANT_PROJECT_SPECIES.SPECIES_ID.eq(SPECIES.ID)),
```

Comment:
These should both be single-value sublists. There can only be one project and one species for a given participant project species.

Changing that ought to fix the failing test.